tests/analysis/losses/risk_calculation/test_risk_calculation.py

```diff
@@ -1,45 +1,158 @@
+    )
+    yield _test_calc_vlh.drop(
+        columns=[
+            "EV1_ma",
+            "EV2_mi",
+            "vlh_business_EV1_ma",
+            "vlh_commute_EV1_ma",
+            "vlh_business_EV2_mi",
+            "vlh_commute_EV2_mi",
+            "risk_vlh_total_default",
+            "risk_vlh_total_triangle_to_null_year",
+            "risk_vlh_total_cut_from_year",
+        ]
+    )
+
+
+@pytest.fixture(name="_expected_factory_results")
+def get_expected_factory_results() -> dict:
+    _expected_factory_results = {
+        RiskCalculationModeEnum.DEFAULT: {
+            "class": RiskCalculationDefault,
+            "_to_integrate": pd.DataFrame(
+                [
+                    [5.869500e06] * 3,
+                    [2.540240e05] * 3,
+                    [1.2899164e06] * 3,
+                    [2.2945832e06] * 3,
+                    [1.459500e06] * 3,
+                    [2.533333e02] * 3,
+                ],
+                columns=[1000, 100, np.inf],
             ),
-            pytest.param(
-                (RiskCalculationModeEnum.CUT_FROM_YEAR, 10000),
-                id=RiskCalculationModeEnum.CUT_FROM_YEAR.name.lower(),
+        },
+        RiskCalculationModeEnum.CUT_FROM_YEAR: {
+            "class": RiskCalculationCutFromYear,
+            "_to_integrate": pd.DataFrame(
+                [
+                    [5.869500e06] * 3,
+                    [2.540240e05] * 3,
+                    [1.2899164e06] * 3,
+                    [2.2945832e06] * 3,
+                    [1.459500e06] * 3,
+                    [2.533333e02] * 3,
+                ],
+                columns=[np.inf, 1000, 500],
             ),
-            pytest.param(
-                (RiskCalculationModeEnum.TRIANGLE_TO_NULL_YEAR, 0),
-                id=RiskCalculationModeEnum.TRIANGLE_TO_NULL_YEAR.name.lower(),
+        },
+        RiskCalculationModeEnum.TRIANGLE_TO_NULL_YEAR: {
+            "class": RiskCalculationTriangleToNullYear,
+            "_to_integrate": pd.DataFrame(
+                [
+                    [5.869500e06] * 3 + [0],
+                    [2.540240e05] * 3 + [0],
+                    [1.2899164e06] * 3 + [0],
+                    [2.2945832e06] * 3 + [0],
+                    [1.459500e06] * 3 + [0],
+                    [2.533333e02] * 3 + [0],
+                ],
+                columns=[np.inf, 1000, 100, 1],
             ),
-        ],
-        name="risk_calculation_info",
-    )
-    def _get_risk_calculation_info(
-        self, request: pytest.FixtureRequest
-    ) -> Iterator[tuple[RiskCalculationModeEnum, int]]:
-        _risk_calculation_info = request.param
-        assert isinstance(_risk_calculation_info[0], RiskCalculationModeEnum)
-        assert isinstance(_risk_calculation_info[1], int)
-        yield _risk_calculation_info
+        },
+        "return_periods": {100, 1000},
+    }
+    yield _expected_factory_results
+
 
+class TestRiskCalculationFactory:
```

Comment:
Each test class should be in a different file,  as well as the fixtures:
- Fixtures to existing / newfile `tests/analysis/losses/risk_calculation/conftest.py` (you don't need to import them afterwards)
- `TestRiskCalculationFactory` to `tests/analysis/losses/risk_calculation/test_risk_calculation_factory.py`